wiki/Beatmaps/en.md

```diff
@@ -1,207 +1,86 @@
 # Beatmaps
 
-## What are "Beatmaps"?
+Beatmaps, or _maps_ for short, are game levels in [osu!](/wiki/osu!_glossary), like the stages in the [DS games](/wiki/Glossary/#ds-games).
+In their most basic form, a beatmap have a song and various [hit objects](/wiki/hit_objects) placed according to the rhythm of the music.
+They can also have a background image, video, and/or [storyboard](/wiki/storyboards).
 
-*Beatmaps listing page: <https://osu.ppy.sh/p/beatmaplist>*
+Creators of beatmaps are called beatmappers or mappers for short.
+Beatmaps are uploaded for everyone to play with using the [BSS](/wiki/BSS), and are then [modded](/wiki/modded).
```

Comment:
"and are then run through an approval process called [modding](...) in order to maintain the high standards of playability